### PR TITLE
Project settings documentation

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Updated user documentation to reflect the changes to the project settings.

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [ ] ~CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
*   [ ] ~Tests added (or description of how to test) for any new features.~
*   [x] User documentation updated for UI or technical changes.

### CODE REVIEW

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looks good just a few things below

File: doc/user/user/project/index.md [hunk 1]
```diff
@@ -142,53 +119,65 @@ Recent activities include adding or modifying project members and adding new sam
...
-### Processing
+### Project Details
+
+Project details include:
+- Project name (required)
+- Project description
+- Project organism
+- As well as the non-modifiable information including project identifier, created date, and the last modified date
+
+![Project settings detail layout.](images/project-settings-details.png)
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/user/project/index.md:134]: Fixed in f179f47

File: doc/user/user/project/index.md [hunk 2]
```diff
@@ -142,53 +119,65 @@ Recent activities include adding or modifying project members and adding new sam
...
 ![Processing tab](images/project-settings-processing.png)
 
-#### Automated pipelines
+#### Project coverage
 
-A project can be setup to automatically trigger the execution of a pipeline on upload of new data.  Any installed pipeline that analyzes individual sample files may be launched on upload (including plugin pipelines).  This setting is enabled on a project-by-project basis and must be enabled by a project **manager**.
+IRIDA can calculate the coverage of uploaded sequencing data for a sample.  To enable this a genome size and expected coverage must be set for a project in the project settings page.
 
-To set up a new automated pipeline, click the *Add Automated Pipeline* button on the processing page.  
+![Coverage](images/project-settings-coverate.png)
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/user/project/index.md:148]: fixed in f179f47

[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/user/project/index.md:148]: Fixed in f179f47

File: doc/user/user/project/index.md [hunk 3]
```diff
@@ -142,53 +119,65 @@ Recent activities include adding or modifying project members and adding new sam
...
+
+Project details include:
+- Project name (required)
+- Project description
+- Project organism
+- As well as the non-modifiable information including project identifier, created date, and the last modified date
+
+![Project settings detail layout.](images/project-settings-details.png)
+
+Managers can edit the modifiable fields by clicking on the edit (pencil) icon within the field.  Updates will be saved immediately after exiting the field.  If you want to cancel the update, just press the `esc` key and the original value will be displayed.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/user/project/index.md]: Updated in e00b996

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**